### PR TITLE
5146: Send bearer token

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/authProvider.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class AuthProvider {
         const params = {
           client_id: oidcClientId,
           redirect_uri: `${appBaseUri}/oauth/callback`,
-          scope: 'openid profile',
+          scope: 'openid profile Organisation',
```
Comment (alibombaye) [inline app/authProvider.js]: Do we need this `Organisation` scope? can you try without and see if everything still runs fine without it.

Comment (kat1906) [reply app/authProvider.js]: `Request failed with status code 401`

File: app/helpers/routerHelper.js [hunk 2]
```diff
@@ -5,3 +5,6 @@ export const withCatch = route => async (req, res, next) => {
     return next(err);
   }
 };
+
+export const extractAccessToken = req => req.session
```
Comment (alibombaye) [inline app/helpers/routerHelper.js]: 😄 nice

File: app/routes.js [hunk 3]
```diff
@@ -29,7 +29,8 @@ export const routes = (authProvider) => {
   router.get('/oauth/callback', authProvider.loginCallback());
 
   router.get('/logout', async (req, res) => {
-    const url = await authProvider.logout();
+    const idToken = req.session && req.session.accessToken && req.session.accessToken.id_token;
```
Comment (alibombaye) [inline app/routes.js]: use that extractAccessToken function here?

File: app/authProvider.js (hunk 1 above)
Comment (alibombaye) [reply app/authProvider.js]: ok let's leave it in then 😂 must need it for the api calls

File: app/routes.js (hunk 3 above)
Comment (kat1906) [reply app/routes.js]: that is the id token but could do something

Comment (alibombaye) [reply app/routes.js]: it's identical ain't it? or am i missing something?

Comment (kat1906) [reply app/routes.js]: `id_token` and `access_token` at the end but I had the helper wrong so changed it now

File: app/pages/dashboard/controller.js [hunk 4]
```diff
@@ -1,26 +1,18 @@
-// import axios from 'axios';
+import axios from 'axios';
 import { getContext } from './contextCreator';
-import { identityServer } from '../../config';
+import { organisationApiUrl } from '../../config';
 import { logger } from '../../logger';
 
-export const getOrgDashboardContext = () => {
-  const endpoint = `${identityServer}/api/v1/Organisations`;
+export const getOrgDashboardContext = async ({ token }) => {
```
Comment (alibombaye) [inline app/pages/dashboard/controller.js]: can we call this accessToken please. Since we have a few tokens in play

File: app/routes.js (hunk 3 above)
Comment (alibombaye) [reply app/routes.js]: don't fancy adding a little helper function for this too. The other one made me happy.

Comment (kat1906) [reply app/routes.js]: I have

Comment (alibombaye) [reply app/routes.js]: yeah just seen it. Sorry it hadn't updated

**Pull request merged**